content/applications/sales/subscriptions/ecommerce.rst

```diff
@@ -0,0 +1,52 @@
+=======================================
+Use subscriptions in the eCommerce shop
+=======================================
+
+Subscription products can be sold in the Odoo *eCommerce* shop just like regular sales products.
+
+However, by default, the eCommerce product page only displays the shortest recurrence period
+listed in the :guilabel:`Time-based pricing` tab of the product form. For example, if a
+subscription product has *monthly* and *yearly* recurrence periods configured, then only the monthly
+price appears on the eCommerce page for that product by default.
+
+To add more recurrence periods to the eCommerce product page, create a *product variant* for each
+recurrence period.
+
+.. seealso::
+   - :doc:`products`
+   - :doc:`Product variants </applications/sales/sales/products_prices/products/variants>`
+
+Configuration
+=============
```

Comment:
[Be specific with headings](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#titles-and-headings)
```suggestion
Create recurrence periods as product variants
=============================================
```